src/Umbraco.Core/Manifest/ManifestParser.cs

```diff
@@ -185,36 +185,43 @@ internal static IEnumerable<PackageManifest> CreateManifests(params string[] man
                 }
 
                 //validate the property editors section
-                var propEditors = deserialized.Properties().Where(x => x.Name == "propertyEditors").ToArray();
+                var propEditors = deserialized.Properties().Where(x => x.Name == ManifestBuilder.PropertyEditorsKey).ToArray();
                 if (propEditors.Length > 1)
                 {
                     throw new FormatException("The manifest is not formatted correctly contains more than one 'propertyEditors' element");
                 }
 
                 //validate the parameterEditors section
-                var paramEditors = deserialized.Properties().Where(x => x.Name == "parameterEditors").ToArray();
+                var paramEditors = deserialized.Properties().Where(x => x.Name == ManifestBuilder.ParameterEditorsKey).ToArray();
                 if (paramEditors.Length > 1)
                 {
                     throw new FormatException("The manifest is not formatted correctly contains more than one 'parameterEditors' element");
                 }
 
                 //validate the gridEditors section
-                var gridEditors = deserialized.Properties().Where(x => x.Name == "gridEditors").ToArray();
+                var gridEditors = deserialized.Properties().Where(x => x.Name == ManifestBuilder.GridEditorsKey).ToArray();
                 if (gridEditors.Length > 1)
                 {
                     throw new FormatException("The manifest is not formatted correctly contains more than one 'gridEditors' element");
                 }
 
+                //validate the dashboards section
+                var dashboards = deserialized.Properties().Where(x => x.Name == "dashboards").ToArray();

```

Comment:
@Shazwazza I guess you can use the `ManifestBuilder.DashboardsKey` variable here as well 😃